zone/client_packet.cpp

```diff
@@ -4476,9 +4473,19 @@ void Client::Handle_OP_ClientUpdate(const EQApplicationPacket *app) {
 			LogError("Can't find boat for client position offset.");
 		}
 		else {
-			cx += boat->GetX();
-			cy += boat->GetY();
+			// Calculate angle from boat heading to EQ heading
+			float theta = fmod(((boat->GetHeading() * 360.0) / 512.0),360.0);
```

Comment:
We should use the versions of these math functions in std namespace at least.

Reply:
Happy to change them. Which ones and what is the benefit?    Just fmod?

Reply:
cos and sin as well. the std namespace versions are type overloaded where the C API ones aren't, which means you will incur float to double conversions (the conversion from EQ headings should be floats as well)

Idealy if we could use the FastCos/FastSin that works on EQ Headings it would be better :P

Reply:
Thanks.  I'll fix it up tomorrow AM and retest.

Reply:
How does this look?  I also reduced the # of calls.